### DISCUSSION TITLE
GeoJSON/Scatterplot layer squares instead of circles

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#5813] [Q&A]
Hi,

I'm trying to display GeoJSON points in first person view. But when I set very small radius, it's displayed as square. Any ideas how can I display circles? Minimap displays  ^^ GeoJSON in MapView.

![image](https://user-images.githubusercontent.com/23722745/119642582-2808ec80-be1b-11eb-96d7-65e9dcb8f6a4.png)

Thanks!

**Comment by sclebal** [comment]
Hey @Radik24, can you give us an example?

**Reply by radoslavirha** [reply]
Hi @Clebal thanks for your answer. I've prepared codepen for you: [url](https://codepen.io/radik24/pen/gOmoxMd).

**Reply by radoslavirha** [reply]
Hi @Clebal do you know where is the problem?

**Reply by Pessimistress** [reply]
Can you open a bug with this codepen? Thank you!

**Reply by sclebal** [reply]
@Radik24 Apologies for the delayed response, I've been out last month. I see you opened an issue, that's great. I also think it's a bug.
